src/ui/card/card.model.ts

```diff
@@ -0,0 +1,5 @@
+export type BaseElevationOptions = 'none' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
```

Comment:
I would either export this to a higher level like `src/models` or rename it to `CardElevation`